docker/Dockerfile

```diff
@@ -20,8 +20,8 @@ RUN mix do deps.get, deps.compile
 ADD . .
 
 ARG COIN
-ARG DISABLE_WRITE_API="false"
-ARG DISABLE_INDEXER="false"
+ARG DISABLE_WRITE_API="true"
+ARG DISABLE_INDEXER="true"
```

Comment:
Inverting these values will break the build of the indexer and web images in production as it changes the default value, if you need to build a docker image with different arguments, you could pass them using `--build-arg DISABLE_WRITE_API=true`

Reply:
Ahh yes! This was a mistake, I was changing things during my local development and left it checked in. I will revert.